### PR TITLE
stability_tester: add bank2 and ledger cases.

### PR DESCRIPTION
**Pull request by disksing**

### CODE REVIEW

File: stability-tester/suite/bank2.go [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+	tx, err := db.Begin()
+	if err != nil {
+		bank2VerifyFailedCounter.Inc()
+		return
+	}
+
+	var tso uint64
+	if err = tx.QueryRow("SELECT @@tidb_current_ts").Scan(&tso); err == nil {
+		log.Infof("SELECT SUM(balance) to verify use tso %d", tso)
```
Comment (siddontang) [inline stability-tester/suite/bank2.go:178]: no return here?

File: stability-tester/suite/bank2.go [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+		return
+	}
+
+	bank2VerifyDuration.Observe(time.Since(start).Seconds())
+
+	expectTotal := (int64(c.cfg.NumAccounts) * initialBalance) * 2
+	if total != expectTotal {
+		log.Errorf("[bank2] bank2_accounts total should be %d, but got %d", expectTotal, total)
+		c.mu.Lock()
+		c.stop = true
```
Comment (siddontang) [inline stability-tester/suite/bank2.go]: use atomic

File: stability-tester/suite/bank2.go [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+		c.wg.Wait()
+		log.Fatalf("[bank2] bank2_accounts total should be %d, but got %d", expectTotal, total)
+	}
+}
+
+// Execute implements Case Execute interface.
+func (c *Bank2Case) Execute(db *sql.DB, index int) error {
+	c.mu.RLock()
+	if c.stop {
+		return errors.New("bank2 stopped")
```
Comment (siddontang) [inline stability-tester/suite/bank2.go]: unlock here or use atomic

File: stability-tester/suite/ledger.go [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+}
+
+// Execute implements Case Execute interface.
+func (c *LedgerCase) Execute(db *sql.DB, index int) error {
+	req := postingRequest{
+		Group:    rand.Int63(),
+		AccountA: fmt.Sprintf("acc%d", rand.Int63n(int64(c.cfg.NumAccounts))),
+		AccountB: fmt.Sprintf("acc%d", rand.Int63n(int64(c.cfg.NumAccounts))),
+		Amount:   1,
+		Currency: "USD",
```
Comment (siddontang) [inline stability-tester/suite/ledger.go]: why USD here

File: stability-tester/suite/bank2.go (hunk 1 above)
Comment (disksing) [reply stability-tester/suite/bank2.go:178]: No need to, it prints the tso for debug only.

File: stability-tester/suite/ledger.go (hunk 4 above)
Comment (disksing) [reply stability-tester/suite/ledger.go]: It's unused, I have removed it.

**Comment by disksing** [comment]
PTAL @siddontang @nolouch

File: stability-tester/config/config.toml [hunk 5]
```diff
@@ -28,14 +28,23 @@ range = 5
 """
 
 [suite]
-names = ["bank", "crud", "block_writer","log"]
+names = ["bank", "bank2", "ledger", "crud", "block_writer","log"]
```
Comment (siddontang) [inline stability-tester/config/config.toml:31]: I think we can remove block_writer and log here. They are not suitable for stability test by default.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by siddontang** [comment]
PTAL @nolouch

**Review by nolouch — APPROVED** [review]
LGTM

**Pull request merged**